python/doc/examples/numerical_methods/iterative_statistics/plot_iterative_moments.py

```diff
@@ -0,0 +1,67 @@
+"""
+Using iterative statistics
```

Comment:
'Estimate statistics iteratively' to be coherent with the rest.

Reply:
Ok. Fixed.